Kota-Yamaguchi/dify
# Feature/add code generator

# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.


# Description

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. Close issue syntax: `Fixes #<issue number>`, see [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for more details.

Fixes 

## Type of Change


# Testing Instructions

You can verify this functionality by following these steps

At First , Launch Workflow/ChatFlow. 
⇨ Create a node code. 
⇨ Click on the Generateicon button in the header 
⇨ Tell the modal that comes up what code you want to write. 
⇨ Apply.
